src/chart.ts

```diff
@@ -635,18 +635,75 @@ export class Chart extends EventEmitter.EventEmitter {
   updateAxes() {
     const instance = this
     const canvas = instance.canvas.merge(instance.canvas.enter)
-    canvas.select('.x.axis').call(instance.meta.xAxis)
-    canvas.select('.y.axis').call(instance.meta.yAxis)
 
-    // updates the style of the axes
+    // center the axes
+    const yMin = this.meta.yScale.domain()[0]
+    const yMax = this.meta.yScale.domain()[1]
+
+    const yMid = (yMax + yMin) / 2
+
+    const yScaleFactor = this.meta.height / (yMax - yMin)
+
+    let yTranslation = yScaleFactor * yMid + this.meta.height / 2
+
+    const xMin = this.meta.xScale.domain()[0]
+    const xMax = this.meta.xScale.domain()[1]
+
+    const xMid = (xMax + xMin) / 2
+
+    const xScaleFactor = this.meta.width / (xMin - xMax)
+
+    let xTranslation = xScaleFactor * xMid + this.meta.width / 2
+
+    if (this.options.xAxis.position === 'sticky') {
```

Comment:
In both cases there's a call to

```
canvas.select('.x.axis').call(instance.meta.xAxis)
```

I believe that call should come before the if statement